src/main/java/com/flowingcode/vaadin/addons/gridexporter/PoiHelper.java

```diff
@@ -0,0 +1,116 @@
+package com.flowingcode.vaadin.addons.gridexporter;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.math.BigInteger;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.xwpf.usermodel.XWPFTableCell;
+import org.openxmlformats.schemas.wordprocessingml.x2006.main.CTTblGridCol;
+import org.openxmlformats.schemas.wordprocessingml.x2006.main.CTTblWidth;
+
+/**
+ * Class for adding support for older versions of Apache POI by using reflection
+ * @author mlopez
+ *
+ */
+public class PoiHelper {
+
+  /**
+   * Tries to set the cell to be blank
+   * @param cell
+   */
+  @SuppressWarnings("deprecation")
+  public static void setBlank(Cell cell) {
+    try {
+      Method setBlank = Cell.class.getMethod("setBlank", Void.class);
```

Comment:
It retrieves `setBlank(Ljava.lang.Void;)` but it calls `setBlank(Lorg.apache.poi.ss.usermodel.Cell;)`, which will always fail.